### PR TITLE
Fix for #2494

### PR DESCRIPTION
**Pull request by solmon4devx**
Restores parameters order as indicated in the manual for the functions of painter class:
   setFont, drawRoundRect, drawArc, drawChord and drawPie

### CODE REVIEW

File: src/modules/objects/KvsObject_painter.cpp [hunk 1]
```diff
@@ -754,15 +754,75 @@ KVSO_END_CONSTRUCTOR(KvsObject_painter)
...
+			c->error(__tr2qs_ctx("The array passed as parameter must contain at least 4 elements","objects"));                              \
+			return false;                                                                                                                   \
+		}                                                                                                                                   \
+		if(__pXOrArray->array()->size() < 6)                                                                                                \
+		{                                                                                                                                   \
+			if(c->params()->count() < 3)                                                                                                    \
+			{                                                                                                                               \
+				QString error=function+__tr2qs_ctx(" requires either an array as first parameter and 2 integers or six integers","objects");\
```
Comment (Arfrever) [inline src/modules/objects/KvsObject_painter.cpp]: `__tr2qs_ctx` is for translation of message into other languages.
You are incorrectly assuming that subject of sentence (in this case, name of function) is at the beginning of sentence in all languages.

File: src/modules/objects/KvsObject_painter.cpp [hunk 2]
```diff
@@ -754,15 +754,75 @@ KVSO_END_CONSTRUCTOR(KvsObject_painter)
...
+		{                                                                                                                                   \
+			c->error(__tr2qs_ctx("One of the geometry array parameters didn't evaluate to an integer","objects"));                          \
+			return false;                                                                                                                   \
+		}                                                                                                                                   \
+	} else {                                                                                                                                \
+		if(c->params()->count() < 6)                                                                                                        \
+		{                                                                                                                                   \
+			QString error=function+__tr2qs_ctx(" requires either an array as first parameter or six integers","objects");                   \
```
Comment (Arfrever) [inline src/modules/objects/KvsObject_painter.cpp]: Same here.

File: src/modules/objects/KvsObject_painter.cpp (hunk 1 above)
Comment (solmon4devx) [reply src/modules/objects/KvsObject_painter.cpp]: I know how to use __tr2qs_ctx I was just trying to stick as close as possible to the style of the rest of the file, but you are right.

**Comment by solmon4devx** [comment]
I think this will not be accepted, so I close the pull request.

**Pull request closed without merging**